docs/Server.md

```diff
@@ -29,37 +32,40 @@ This option also applies when the
 <code><b>http2</b></code>
 </a> option is set.
 
-+ Default: `null`
+* Default: `null`
```

Comment:
Why is this file different? The other got changed to `-` for lists, but this one to `*`?